src/components/Footer.jsx

```diff
@@ -1,7 +1,40 @@
+// import Image from "next/image";
+// import Logo from "@/public/Logo.webp";
+import { BsDiscord } from "react-icons/bs"; //  for discord
+import { BsInstagram } from "react-icons/bs"; //  for insta
+import { BsEnvelope } from "react-icons/bs"; // for mailing
+
 const Footer = () => {
   return (
-    <div>
-      <p>Footer</p>
+    <div className="box-content h-32 w-128 bg-gradient-to-r from-lime-950 via-lime-900 to-lime-800">
```

Comment:
change to w-full, don't use a specific number because it needs to be responsive to different screen sizes. Also remove box-content, not needed